.github/workflows/test.yaml

```diff
@@ -36,8 +36,18 @@ jobs:
         kind version
         kind create cluster
         kind export kubeconfig
+    - name: Install the rukpak k8s provisioner
+      run: |
+        make generate
+        kubectl create namespace rukpak-system
+        kubectl apply -f manifests
+
+        # Note: run the provisioner in the background so we don't block
+        # running ginkgo tests in the foreground.
+        # TODO: remove this hack when we have k8s provisioner manifests
+        # and more fleshed out CI pipeline that builds/pushes the unpack
+        # and provisioner image(s) so we can reference them at runtime.
+        nohup go run provisioner/k8s/main.go &
```

Comment:
Nit/follow-up only: It would be great to get this into a make target such that:
1. e2e's are easily runnable locally
2. e2e setup and environment is fully encapsulated by the Makefile so that local runs are identical to CI runs (e.g. exact versions of kind, the cluster version, and envtest)

Ideal it would be:
- Locally I can run `make e2e`
- This workflow is just `checkout`, `setup-go`, `make e2e`

Reply:
@joelanford I took at stab at doing this in https://github.com/operator-framework/rukpak/pull/85.